GG/src/dialogs/ColorDlg.cpp

```diff
@@ -611,15 +611,14 @@ void ColorDlg::Init(const std::shared_ptr<Font>& font)
         }
     }
 
-    using boost::lexical_cast;
     m_sliders_ok_cancel_layout = new Layout(m_pickers_layout->Right() + 5, Y0, X(150), Y((25 + 5) * 8 - 5),
                                             9, 3, 0, 5);
     m_sliders_ok_cancel_layout->SetMinimumColumnWidth(0, X(15));
     m_sliders_ok_cancel_layout->SetMinimumColumnWidth(1, X(30));
     m_sliders_ok_cancel_layout->SetColumnStretch(2, 1);
     m_slider_labels.push_back(style->NewTextControl(style->Translate("R:"), font, m_text_color, FORMAT_RIGHT));
     m_sliders_ok_cancel_layout->Add(m_slider_labels.back(), 0, 0);
-    m_slider_values.push_back(style->NewTextControl(lexical_cast<std::string>(static_cast<int>(color.r)),
+    m_slider_values.push_back(style->NewTextControl(std::to_string(static_cast<int>(color.r)),
```

Comment:
The cast shouldn't be required and does a unsigned to signed conversion.

GG::Clr holds `unsigned char` as color channels.  Also std::to_string only accepts int, long, long long and the unsigned counterpart as integer inputs.  This should implicit cast uchar to uint but better check if the control still displays the correct value and not a character.